ansible/roles/cluster_start/tasks/main.yml

```diff
@@ -0,0 +1,11 @@
+---
```

Comment:
We should better remove this implementations by now.
Let's only focus in the `checks` part